modules/apps/web-experience/layout/layout-impl/src/main/java/com/liferay/layout/internal/exportimport/staged/model/repository/StagedLayoutSetStagedModelRepository.java

```diff
@@ -241,7 +241,7 @@ public StagedLayoutSet updateStagedModel(
 		if (layoutSetSettings) {
 			existingLayoutSet = _layoutSetLocalService.updateSettings(
 				existingLayoutSet.getGroupId(),
-				existingLayoutSet.isPrivateLayout(),
```

Comment:
`existingLayoutSet` is originally from the site template and site template is always created using the private layout, so this is always true. But when we import the site template, it may be imported to the public or private layout.